ports/rpclib/portfile.cmake

```
@@ -3,23 +3,24 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO rpclib/rpclib
-    REF v2.2.1
-    SHA512 a63c6d09a411fb6b87d0df7c4f75a189f775ff0208f5f1c67333a85030a47efe60d5518e5939e98abc683a6063afb6cfed51f118f594a2a08be32330a9691051
+    REF v2.3.0
+    SHA512 9C65AE5D000CD66E39498667DC826A4E8A2C28EB4138CFFE3AA09D5DDC22575551B9FA6CE9D951343213B9583F947899CB18B58130AEAEE36DAB5881EA496272
     HEAD_REF master
 )
 
 vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
+    SOURCE_PATH "${SOURCE_PATH}"
     DISABLE_PARALLEL_CONFIGURE
     PREFER_NINJA
 )
 
 vcpkg_install_cmake()
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/rpclib)
+vcpkg_fixup_cmake_targets(CONFIG_PATH "lib/cmake/rpclib")
```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/rpclib)
```
Quotes are not needed for a relative path.

---

Through internal discussion, adding the quotes or not are both okay, so it's unnecessary to update this.